Makefile.am

```diff
@@ -49,15 +49,16 @@ dist-hook:
 # compatibility links will be removed in the next major release.
 if CONFIG_KERNEL
 install-data-hook:
+	rm -rf $(DESTDIR)$(prefix)/src/spl-$(VERSION) && \
```

Comment:
do we really need to `rm`? or is it better to `mkdir -p` ?

Reply:
My first thought was also `mkdir -p`, but @behlendorf suggested `rm`.  This also seems like a cleaner way to transition from separate-spl to merged-spl.

Reply:
I thought the separation should be managed by $(VERSION), in any case, though it is a bit rude, it will work.

Reply:
Yup, it's not pretty but it should be functional while we transition.

Reply:
@ryao I'm OK with either solution, but since this one was just merged we'll need a new PR to switch to `-p` if that's strongly preferred.